packages/analytics-js/src/components/core/Analytics.ts

```diff
@@ -312,20 +323,20 @@ class Analytics implements IAnalytics {
   /**
    * Trigger onLoaded callback if any is provided in config
    */
-  onLoaded() {
+  onInitialized() {
     // Process any preloaded events
     this.processDataInPreloadBuffer();
 
+    // Execute onLoaded callback if provided in load options
+    if (isFunction(state.loadOptions.value.onLoaded)) {
+      (state.loadOptions.value.onLoaded as OnLoadedCallback)((globalThis as any).rudderanalytics);
```

Comment:
(globalThis as any).rudderanalytics will not provide the correct instance in case more that one analytics instance is available.

What is the issue we attempt to fix and we changes the argument value here?

Reply:
Correct.
The callback is supposed to forward to the analytics instance. Earlier, we were sending `this` which is the Analytics class object.

Reply:
How do we solve the specific analytics instance problem?